app/Models/User.php

```diff
@@ -2,32 +2,24 @@
 
 namespace App\Models;
 
-use Illuminate\Auth\Authenticatable;
-use Illuminate\Contracts\Auth\Access\Authorizable as AuthorizableContract;
-use Illuminate\Contracts\Auth\Authenticatable as AuthenticatableContract;
-use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
-use Laravel\Lumen\Auth\Authorizable;
 
-class User extends Model implements AuthenticatableContract, AuthorizableContract
+/**
+ * App\Models\User
+ *
+ * @property string $username
+ * @property string $email
+ * @property string $role
+ * @property int $bigcommerce_user_id
+ * @property int $store_id
+ */
+class User extends Model
 {
-    use Authenticatable, Authorizable, HasFactory;
+    protected $table = 'users';
```

Comment:
That is optional. You only really need to manually set this, if your model was called `A` but your table is called `B`.